randovania/games/prime3/json_data/header.json

```diff
@@ -1115,6 +1115,10 @@
             "Event155": {
                 "long_name": "Imperial Hall Grapple Lasso Walls",
                 "extra": {}
+            },
+            "ShipGrappleAcquired": {
+                "long_name": "SkyTown East - Ship Grapple Acquired",
```

Comment:
I'd rename this event to not mention the vanilla item. Just say Pickup instead

Reply:
Simple change